text/3595-move-crates-io-team-under-dev-tools.md

```diff
@@ -0,0 +1,82 @@
+
+For the purposes of actual decision making, the Crates.io subteam retains all decision-making power with regard to crates.io related issues (i.e., this proposal does not change who makes any particular decision and is purely a change in Council representation). This may change over time should the Dev tools team choose to structure itself in a different way.
+
+# Practicalities
+
+Once this proposal is accepted, the Crates.io team will move to be a subteam of Dev tools. The Dev tools team does not change its top-level form.
+
+The Dev tools team's Council representative would continue to serve on the Council while the Crates.io representative would immediately stop counting as a representative for all purposes.[^plan]
+
+[^plan]: It is currently the unofficial plan that Carol Nichols will step down in her role as the Crates.io representative, and Eric Huss would take over as the rep, but this would be made official after the merger through internal Dev tools team process.
+
+# Alternatives
+
+## Merge Crates.io into Infrastructure
+
+Crates.io uses infrastructure such as Fastly, CloudFront, S3, Heroku, GitHub, and other services that are managed by the Infrastructure team. There's certainly an argument to be made that the Crates.io team belongs there, however, the docs.rs team is in a similar situation and they are a subteam of Dev tools.
+
+## Be a subteam of both Dev tools and Infra
+
+The Types team provides precedence for this; technically the Types team is a subteam of both the Compiler and Lang teams. However, the teams repo doesn't really support multiple inheritance. The Crates.io team would like to cultivate a closer relationship with the Cargo team especially, and thinks the relationship with the Infra team could be continued in the current manner.
+
+## Creating a new team
+
+There are aspects of the Crates.io Team's purview that are more policy decisions than they are implementation details, such as what information Crates.io should surface about each crate, how to handle different crate ownership situations, or what browsers the site should support.
+
+There could be a new team dedicated to policy questions such as these and other questions that have currently come up to the Leadership Council. People who enjoy thinking about and discussing policies might enjoy being on this team and not being responsible for implementing the policies. People implementing the policies might enjoy not being responsible for creating the policies.
+
+However, it isn't clear if a policy team would be feasible and desirable. If so, this division could be done as a future enhancement; this RFC does not prevent such.
+
+# Prior Art
+
+Many thanks to Ryan Levick's [RFC 3533], much of which was copy-pastaed into this one. ❤️
```

Comment:
Small typo I think (but we can keep it because who doesn't like pasta? :smile:):

```suggestion
Many thanks to Ryan Levick's [RFC 3533], much of which was copy-pasted into this one. ❤️
```

Reply:
I believe this is deliberate. https://en.wikipedia.org/wiki/Copypasta

Reply:
TIL.